spyder/utils/tests/test_iofuncs.py

```diff
@@ -13,13 +13,25 @@
 import numpy as np
 import spyder.utils.iofuncs as io
 
-__location__ = os.path.realpath(os.path.join(os.getcwd(),
+LOCATION = os.path.realpath(os.path.join(os.getcwd(),
                                              os.path.dirname(__file__)))
```

Comment:
Please align this line correctly to be at the same level of `os.getcwd()` above.